### ISSUE TITLE
Extra Backslashes in URL Field

### ISSUE THREAD

**Issue by chenle02** [FlamingTempura/bibtex-tidy#415]
The input:

```
@incollection {MR4175746,
    AUTHOR = {Bourgain, Jean and Demeter, Ciprian and Kemp, Dominique},
     TITLE = {Decouplings for real analytic surfaces of revolution},
 BOOKTITLE = {Geometric aspects of functional analysis. {V}ol. {I}},
    SERIES = {Lecture Notes in Math.},
    VOLUME = {2256},
     PAGES = {113--125},
 PUBLISHER = {Springer, Cham},
      YEAR = {[2020] \copyright 2020},
      ISBN = {978-3-030-36020-7; 978-3-030-36019-1},
   MRCLASS = {42B10 (53A05)},
  MRNUMBER = {4175746},
MRREVIEWER = {Bochen\ Liu},
       DOI = {10.1007/978-3-030-36020-7\_7},
       URL = {https://doi.org/10.1007/978-3-030-36020-7_7},
}

```

the output:

```
@incollection{MR4175746,
  author        = {Bourgain, Jean and Demeter, Ciprian and Kemp, Dominique},
  title         = {Decouplings for real analytic surfaces of revolution},
  booktitle     = {Geometric aspects of functional analysis. {V}ol. {I}},
  series        = {Lecture Notes in Math.},
  volume        = {2256},
  pages         = {113--125},
  publisher     = {Springer, Cham},
  year          = {[2020] \copyright 2020},
  isbn          = {978-3-030-36020-7; 978-3-030-36019-1},
  mrclass       = {42B10 (53A05)},
  mrnumber      = {4175746},
  mrreviewer    = {Bochen\ Liu},
  doi           = {10.1007/978-3-030-36020-7\_7},
  url           = {https://doi.org/10.1007/978-3-030-36020-7\_7}
}


```

**Comment by chrisgrieser** [comment]
I also noticed those `\` being added and making most values invalid. The only way to disable that seems to be to pass `--no-escape` (with the downside of losing the otherwise useful escaping feature)

**Comment by FlamingTempura** [comment]
Fixed in v1.14.0
